lib/register/action.js

```diff
@@ -53,7 +53,7 @@ function registerAction(options) {
   if (typeof options.do !== 'function')
     throw new Error('do must be a function');
   // Don't throw an error for not having an undo action,
-  // we will just throw a warning when a user performs a clean.
+  // we will throw only warning when a user performs a clean.
```

Comment:
This sounds a bit weird removing the "a"

Reply:
OK. Made change here.